package.json

```diff
@@ -17,7 +17,7 @@
     "postpublish": "git add packages/*.md packages/*/index.d.ts && git commit -m 'Publish: automate readme, type definitions and changelog update' --no-verify && git push origin master -f --no-verify",
     "prepare": "yarn cleanup && lerna run pretranspile && yarn transpile && yarn generateReadMe && yarn generateTypeDefinitions",
     "prepublishOnly": "yarn generateChangelog",
-    "prettier": "prettier \"**/*.+(js|json|yml|css|scss)\"",
+    "prettier": "prettier \"packages/**/*.+(js|json|yml|css|scss)\"",
```

Comment:
a small comment, we already have prettierignore set up, do we really need this to be run in the packages folder?

Reply:
I'm not entirely sure what other files/libraries are being scanned apart from those ignore, but in my case read files that are not relevant plus the one in packages, making it slower, I can remove it, but is speeding the process by targeting exactly the files that we care.